blob/azureblob/azureblob_test.go

```diff
@@ -53,6 +55,7 @@ import (
 const (
 	bucketName  = "go-cloud-bucket"
 	accountName = AccountName("gocloudblobtests")
+	cloudEnv    = CloudEnvironment("")
```

Comment:
I think you can leave this out as well.

Reply:
Sure (also to all the below changes). Something made me squeamish about adding this option without any automated way to test it, but if you're okay with it, I can remove all of these changes to the testing file.